site/_quarto.yml

```diff
@@ -37,9 +37,7 @@ website:
           file: guide/supported-models.qmd
         - text: "---"
         - text: "{{< fa code >}} CODE SAMPLES"
-        - text: "{{< fa book >}} QuickStart · `Customer Churn` · `Binary Classification`"
-          file: notebooks/quickstart_customer_churn_full_suite.ipynb
```

Comment:
The QuickStart doesn't render the site sidebar correctly when embedded in a menu, so I removed it.